### ISSUE TITLE
Magpie error when alt+f11 on games

### ISSUE THREAD

**Issue by franvmm** [Blinue/Magpie#66]
When i try to do the command and the settings: FSR + winrt + injection at runtime/ no injection MAGPIE returns error and cursor dissapears, so you don't know where it is.
Excamples of games with this issue:
Raceroom racing experience
WRC9

**Comment by Blinue** [comment]
Does the error only happens in this game?

**Comment by franvmm** [comment]
When pressing the command, and a yellow border stays while magpie crash

**Comment by Blinue** [comment]
This is a bug of Magpie, but I can't reproduce it because I don't have these games. Please wait for the next version, which will support logging.

**Comment by franvmm** [comment]
[CursorHook.log](https://github.com/Blinue/Magpie/files/7076370/CursorHook.log)
[Magpie.log](https://github.com/Blinue/Magpie/files/7076371/Magpie.log)
 attached logs for this issue

**Comment by Blinue** [comment]
Thanks for the feedback.

Since the Runtime module has no logs for some reason, I can only tell that it's a rendering failure. It could be a bug in Windows.Graphics.Capture, because there are no other similar reports.

**Comment by ribolovac510** [comment]
I have noticed this issue on an old pinball game that I have called Full Tilt! Pinball. It has a launcher for three different pinball tables called FULLTILT.EXE, and the pinball tables have their own executables in separate folders. Magpie is able to upscale the launcher when I press Alt+F11 but not the tables that are run through it. If I run any table from their own .exe, Magpie is then able to upscale them. I've tried changing the Capture Mode options and selecting the Injection at Runtime option but it didn't help. Attached is a screenshot of the game's files and a log file.

![Screenshot](https://user-images.githubusercontent.com/74596991/131504184-25820f3a-0fe7-4838-b1c8-85849050d935.png)
[Magpie.log](https://github.com/Blinue/Magpie/files/7083969/Magpie.log)
